File: Java/fibonacci.java
/**
	Description: Display fibonacci series upto n, taken as input
	Fibonacci: starts from 0,1 and next number of the series is obtained by adding the previous two terms
*/

import java.util.Scanner;
public class fibonacci {

	// fibo method calculates the next term in the series and runs till n
	public static void fibo(int n)
	{
		// variable initialization
		int first = 0, second = 1, next = 0;
		// displaying the first and second term
		System.out.println(first + "\n" + second);

		for (int i=0; i<n; i++)
		{
			// finding the next term by adding the previous two terms
			next = first + second;
			System.out.println(next);
			// updating first and second
			first = second;
			second = next;
		}
	}

	// main method
	public static void main(String[] args)
	{
		int n;
		Scanner sc = new Scanner(System.in);
		// input range	
		System.out.println("Enter the range of the fibonacci series: ");
		n = sc.nextInt();
		// calling fibo with n as argument
		fibo(n);
	}
}

File: C/fibonacci.c
#include<stdio.h>
    int main()
{
    int n,k;
    long int aray[20];

    printf("Enter the number range:\n");
    scanf("%d",&n);
    aray[0]=-1;
    aray[1]=1;
    printf("\n FIBONACCI SERIES: \n");
    for(k=2;k<=n+1;k++)
    {
      aray[k]=aray[k-1]+aray[k-2];
      printf(" %ld",aray[k]);
    }
  return 0;
}